Vagrantfile

```diff
@@ -0,0 +1,32 @@
+Vagrant.configure("2") do |config|
+config.vm.box = "hashicorp/bionic64"
+ 
+config.vm.provision "shell", privileged: false, inline: <<-SHELL
+	sudo apt-get update
+ 
+	sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+	git clone https://github.com/pyenv/pyenv.git /$HOME/.pyenv
+
+	echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
+	echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile
+    echo 'eval "$(pyenv init -)"' >> ~/.profile
+    pyenv install 3.8.5
+	pyenv global 3.8.5
+	eval "$(pyenv init -)"
+	
+
+	curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python  
+SHELL
+
+	config.trigger.after :up do |trigger|
+    trigger.name = "Launching App"
+    trigger.info = "Running the TODO app setup script"
+    trigger.run_remote = {privileged: false, inline: "
+      cd /vagrant
+      poetry install
+	  poetry run flask run --host 0.0.0.0
```

Comment:
This currently confuses Vagrant's VM management because "vagrant up" will hang until you kill it.

You could either use gunicorn (stretch goal) or you can start this command as a background process with something like `nohup poetry run flask run --host 0.0.0.0 > logs.txt 2>&1 &`